README.md

```diff
@@ -1,229 +1,144 @@
 
-2. Processor - 2 Core/vCPU 1.1GHz or higher
+To learn more about WSO2 Identity Server please visit [WSO2 Identity Server website](https://wso2.com/identity-server/)
 
-3. Java SE Development Kit 11
 
-4. The Management Console requires full Javascript enablement of the Web browser.
-
-5. To build WSO2 Identity Server from the Source distribution, it is also necessary that you have Maven 3 or later.
-
-For more details see
-   * https://is.docs.wso2.com/en/7.1.0/deploy/get-started/install/ (For IS 7.1.0 version)
-
-Project Resources
-=================
-
-* Home page          : http://wso2.com/products/identity-server
-* Library            : https://wso2.com/library/identity-and-access-management
-* Wiki 
-    * IS 7.1.0 version     : https://is.docs.wso2.com/en/7.1.0/
-* Issue Tracker      : https://github.com/wso2/product-is/issues      
-* Forums             : http://stackoverflow.com/questions/tagged/wso2is/
-* IAM Developer List: iam-dev@wso2.org
-
-
-Building the distribution from source
-=========================================
-
-1. Install Java SE Development Kit 11
-2. Install Apache Maven 3.x.x(https://maven.apache.org/download.cgi#)
-3. Get a clone from https://github.com/wso2/product-is.git or download the source 
-4. Run the one of the below maven commands from product-is directory, 
-    - `mvn clean install` (To build the binary and source distributions with the tests)
-    - `mvn clean install -Dmaven.test.skip=true` (To build the binary and source distribution, without running any of 
-    the unit/integration tests)
-5. You can find the binary distribution in `product-is/modules/distribution/target` directory.
-
-    
-Installation and Running
-========================
+> ##### Looking for a Hosted Identity Solution?
+> Try **[Asgardeo](https://wso2.com/asgardeo/)** free of charge – a public identity cloud solution (iDaaS) offering, built on the same technology core as **WSO2 Identity Server**.
 
-1. Extract the downloaded/built binary distribution zip file
-2. Run the wso2server.sh or wso2server.bat file in the /bin directory
-3. Once the server starts, point your Web browser to https://localhost:9443/carbon/
-4. Developer and Administrator Console is available at https://localhost:9443/console
-5. End User Portal is available at https://localhost:9443/myaccount
-6. For more information, see the Installation Guide
 
+Why WSO2 Identity Server?
+=========================
+Whether you’re securing apps for external consumers, internal workforce, business customers, or even API access, WSO2 Identity Server provides the modern, full-featured identity and access foundation you need. 
 
-WSO2 Identity Server Distribution Directory Structure
-==============================================
+Customers prefer Identity Server for its developer-centric design, extensive standards support, freedom to deploy however needed, and massive scalability.
```

Comment:
I'm not sure if including a marketing pitch in the README of the open source repository is entirely appropriate. Since we've already linked to the Identity Server website, I feel it might not be necessary to reiterate that information here. just my personal opinion, of course.

Reply:
+1. @sagara-gunathunga WDYT ?